Text/CustomEntitySearch/WordLinker.cs

```diff
@@ -10,14 +11,37 @@ internal class WordLinker
     {
         public WordLinker(string executingDirectoryPath)
         {
-            string json = File.ReadAllText($"{executingDirectoryPath}\\words.json");
-            Words = new List<string>(
-                JsonConvert.DeserializeObject<List<string>>(json));
+            string json = File.ReadAllText($"{executingDirectoryPath}\\Users\\t-neja\\azure-search-power-skills\\Text\\CustomEntitySearch\\words.json");
```

Comment:
What is this path?

Reply:
Sorry, with my version of testing, I needed to use this path. Will delete.